openpype/hosts/resolve/startup.py

```diff
@@ -0,0 +1,40 @@
+import os
+
+# Importing this takes a little over a second and thus this means
+# that we have about 1.5 seconds delay before the workfile will actually
+# be opened at the minimum
+import openpype.hosts.resolve.api
+
+
+def launch_menu():
+    from openpype.pipeline import install_host
+    print("Launching Resolve OpenPype menu..")
+
+    # Activate resolve from openpype
+    install_host(openpype.hosts.resolve.api)
```

Comment:
Shouldn't this happen in `main` before anything else?

Reply:
I mostly _tried_ to delay it as much as possible since it's relatively slow and Resolve's startup UI is active during that time. So a user might be inclined to click to open a project even though the "open last workfile" bit might still.

It's good to note that this launch script runs in its own separate process - it doesn't influence the main Resolve process. So I'd recommend only ever triggering host install if whatever we're doing actually requires an installed host to begin with.

Reply:
I though this scipt is actually triggered all the time, but it happens only if workfile should be opened right?

Reply:
The startup script triggers on every start of Resolve - but workfile opening only happens if it's enabled, same for the startup menu start now as well.